cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -98,6 +98,52 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool, auditWa
 	} else {
 		fmt.Fprintln(out, "  Applying", len(policies)+len(vaps), pluralize.Pluralize(len(policies)+len(vaps), "policy", "policies"), "to", len(uniques), pluralize.Pluralize(len(uniques), "resource", "resources"), "...")
 	}
+	policyToUnknownRules := make(map[string]string)
+	unknwonPolicies := make(map[string]string)
```

Comment:
Why do you use maps? You are using this as follows:
```
unknwonPolicies[res.Policy] = res.Policy
```

Instead, you can define it as a list of strings.